maps/tether/tether-05-station1.dmm

```diff
@@ -12948,18 +13403,53 @@
 /turf/simulated/floor/tiled,
 /area/storage/tools)
 "awV" = (
-/obj/effect/floor_decal/rust,
+/obj/machinery/suit_storage_unit/standard_unit,
+/turf/simulated/floor/tiled/dark,
+/area/bridge/secondary/teleporter)
+"awW" = (
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
 /obj/machinery/atmospherics/pipe/simple/hidden/supply,
-/obj/structure/disposalpipe/segment,
-/turf/simulated/floor,
-/area/maintenance/station/spacecommandmaint)
-"awY" = (
-/obj/structure/bed/chair{
-	dir = 1
+/obj/structure/cable/green{
+	d1 = 1;
+	d2 = 4;
+	icon_state = "1-4"
 	},
-/turf/simulated/floor/wood/broken,
-/area/maintenance/station/spacecommandmaint)
+/obj/machinery/light_switch{
+	dir = 2;
+	name = "light switch ";
+	pixel_x = 20;
+	pixel_y = 30
+	},
+/obj/effect/floor_decal/steeldecal/steel_decals4{
+	dir = 9
+	},
+/obj/effect/floor_decal/steeldecal/steel_decals4{
+	dir = 4
+	},
+/turf/simulated/floor/tiled,
+/area/bridge/secondary/teleporter)
+"awX" = (
+/obj/effect/floor_decal/industrial/warning{
+	icon_state = "warning";
+	dir = 4
+	},
+/obj/machinery/power/apc{
+	dir = 1;
+	name = "north bump";
+	pixel_x = 0;
+	pixel_y = 24
+	},
+/obj/structure/cable/green{
+	d2 = 8;
+	icon_state = "0-8"
+	},
+/turf/simulated/floor/tiled,
+/area/bridge/secondary/teleporter)
+"awY" = (
+/obj/structure/table/standard,
+/obj/item/weapon/hand_tele,
```

Comment:
My main point of contention here would be the presense of a second Hand Tele. I still think that in terms of those, only one should be available.

Reply:
```suggestion

```

Can just remove it if that's desirable.

Reply:
I'd like to hear the admins' input regarding that obj and whether or not it should be kept in. Purrsonally, I don't see much of an issue with it being there. It can be used for televore and more of that can be good for scenes as a whole.